docs/development/bump-node-major.md

```diff
@@ -0,0 +1,18 @@
+# How to bump Renovate to next NodeJS lts release
+
+## Add new NodeJS version
+
+- Add new versions via `package.json>engines>node` and `package.json>engines-next>node`
+- Update node at [local-development](./local-development.md)
+- Update node versions at Github Actions
+
+## Deprecate old NodeJS version
+
+- Deprecate old lts via `package.json>engines-next>node`
+- Update node [local-development](./local-development.md)
+- Remove node versions at Github Actions
+
+## Remove old NodeJS version
+
+- Update `package.json>engines>node`
+- Mark PR as `BREAKING`
```

Comment:
Reading this, it's not clear how to `Mark PR as BREAKING`. 😕 
Do you put `BREAKING feat:` in the PR title, or do you put `BREAKING` in the commit message?

Please explain precisely how to mark the PR breaking. 😄

Reply:
Add label `breaking` and text `BREAKING CHANGE: ...`  🙃

Reply:
@HonkingGoose Can you make a suggestion now? 🙃

Reply:
```suggestion
- Mark PR as `BREAKING` by:
  - Adding the label `breaking` to the PR
  - Putting the text `BREAKING CHANGE:` in the PR title
```

Let's see if this renders properly. 😄

Reply:
`BREAKING CHANGE: <breaking cause>` comes to pr content

Reply:
@viceice I've updated my suggestion.